data/maps/SP_Scenarios/The_Green_Plateau.wmf/scripting/texts.lua

```diff
@@ -48,21 +48,25 @@ obj_defeat_jomo = {
 -- Messages to the User
 -- =====================
 
+mb_w = messagebox_w_step(-1)
+mb_h = messagebox_h_step(-2)
+mb_y = styles.get_size("map_plateau_message_pos_y")
+
 briefing_1_the_forbidden_island = {
    title = _("First Briefing"),
-   width = 400,
-   height = 300,
-   posy = 1,
```

Comment:
Actually I think this one's just the map author's preference, so `map_plateau_message_pos_y` could be left out, because `campaign_message_box_top_pos_y`  can be changed by a customised theme addon for anyone who doesn't like the default.